bugbug/bug_snapshot.py

```diff
@@ -870,8 +875,8 @@ def get_inconsistencies(bugs):
         try:
             rollback(bug, do_assert=True)
         except Exception as e:
-            print(bug["id"])
-            print(e)
+            logger.exception(bug["id"])
+            logger.exception(e)
```

Comment:
```suggestion
```
This line is not needed. The `logger.exception()` method will automatically log the exception itself.